arm-sql/2014-04-01/swagger/auditingPolicies.json

```diff
@@ -0,0 +1,410 @@
+                    "description": "Gets the retention in days of the Azure SQL Database auditing policy."
+                },
+                "auditLogsTableName": {
+                    "type": "string",
+                    "description": "Gets the name of the table audit logs are written to in the Azure SQL Database auditing policy."
+                },
+                "fullAuditLogsTableName": {
+                    "type": "string",
+                    "description": "Gets the full name of the table audit logs are written to in the Azure SQL Database auditing policy."
+                }
+            },
+            "description": "Represents the properties of an Azure SQL auditing policy."
+        },
+        "ServerAuditingPolicy": {
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/ServerAuditingPolicyProperties",
+                    "description": "Represents the properties of the resource."
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                }
+            ],
+            "description": "Represents an Azure SQL Server auditing policy."
+        },
+        "Resource": {
+            "description": "Resource properties",
+            "properties": {
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource name"
+                },
+                "id": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource Id"
+                },
+                "type": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource type"
+                },
+                "location": {
+                    "type": "string",
+                    "description": "Resource location"
+                },
+                "tags": {
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "string"
+                    },
+                    "description": "Resource tags"
+                }
+            },
+            "required": [
+                "location"
+            ],
+            "x-ms-azure-resource": true
+        },
+        "SubResource": {
+            "description": "Subresource properties",
+            "properties": {
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource name"
+                },
+                "id": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource Id"
+                }
+            },
+            "x-ms-azure-resource": true
```

Comment:
Please do. 

---
In reply to: [80103351](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80103351) [](ancestors = 80103351)